src/doc/src/reference/workspaces.md

```diff
@@ -127,6 +133,7 @@ used:
 [workspace]
 members = ["path/to/member1", "path/to/member2", "path/to/member3/*"]
 default-members = ["path/to/member2", "path/to/member3/foo"]
+resolver = "2"
```

Comment:
For me, adding `resolver = "2"` to all examples clutters them ,distracting from what we are trying to teach

Reply:
removed from all the examples but virtual workspace